templates/zerver/integrations.html

```diff
@@ -958,6 +958,11 @@
           <pre><code>HUBOT_ZULIP_BOT=hubot-bot@example.com HUBOT_ZULIP_API_KEY=your_key bin/hubot --adapter zulip --name "myhubot"</code></pre>
 
         </li>
+
+        <li>To run Hubot with a self-hosted version of Zulip, use::
+          <pre><code>HUBOT_ZULIP_SITE={{ external_api_path_subdomain }} HUBOT_ZULIP_BOT=hubot-bot@example.com HUBOT_ZULIP_API_KEY=your_key bin/hubot -a zulip</code></pre>
+
+        </li>
```

Comment:
I don't think that this actually works; skimming the code of the hubot plugin (https://github.com/zulip/hubot-zulip), it seems to expect to not have the trailing `/api` and to have the `https://` part included... we might want to fix this by patching zulip-node to handle an extra trailing `/api`...